src/js/tech/tech.js

```diff
@@ -498,6 +498,26 @@ Tech.withSourceHandlers = function(_Tech){
     handlers.splice(index, 0, handler);
   };
 
+  /*
+   * Check if the tech can support the given type
+   * @param  {String} type    The mimetype to check
+   * @return {String}         'probably', 'maybe', or '' (empty string)
+   */
+  _Tech.canPlayType = function(type){
```

Comment:
This means that only sourcehandlers will get the `canPlayType` method, I believe. We probably want all techs to have a canPlayType. Probably need to add a `canPlayType = () => ''` to the prototype of `Tech`.